filter.md

```diff
@@ -0,0 +1,217 @@
+    show_vanilla = no
+
+    # Whether to report about problems in mods loaded via the load_mod sections.
+    # Optional boolean value, defaults to no.
+    show_loaded_mods = no
+    
+    # You can have at most 1 trigger block.
+    trigger = {
+        # The root trigger block is an AND-gate, just like in paradox script.
+    }
+}
+```
+
+## Triggers
+
+This is the list of valid triggers that can be used in the `trigger` block.
+
+### always
+
+- `always = yes` matches all reports
+- `always = no` matches no reports
+
+### Logic gates
+
+You can use any of the following logic gates. The `NOT` gate must contain exactly one trigger. All other gates take zero
+or more triggers.
+
+- `NOT = { <trigger> }`
+- `AND = { <trigger> <trigger> ... }`
+    - Matches reports that match all children.
+    - An `AND` block without children is the same as `always = yes`.
+- `OR = { <trigger> <trigger> ... }`
+    - Matches reports that match at least one child.
+    - An `OR` block without children is the same as `always = no`.
+- `NAND = { <trigger> <trigger> ... }`
+    - This is the same as writing `NOT = { AND = { } }`
+- `NOR = { <trigger> <trigger> ... }`
+    - This is the same as writing `NOT = { OR = { } }`
+
+### Severity
+
+Severity is a measure of how serious the problem being reported about is. For example; using three spaces of indentation
+instead of four is an extremely minor problem that wouldn't impact the functionality of your mod at all. It would be a
+very low severity. Conversely, a problem that causes the game to crash on start-up should receive the highest severity.
+
+These are the current severity levels:
+
+- `Tips` These are things that aren't necessarily wrong, but there may be a better, more idiomatic way to do it. This may also include performance issues.
+- `Untidy` This code smells. The player is unlikely to be impacted directly, but developers working on this codebase will likely experience maintenance headaches.
+- `Warning` This will result in glitches that will noticeably impact the player's gaming experience. Missing translations are an example.
+- `Error` This code probably doesn't work as intended. The player may experience bugs.
+- `Fatal` This is likely to cause crashes.
+
+You can add the severity trigger like so:
+
+- `severity == Tips` (Only match tips.)
+- `severity >= Warning` (Match Warnings or higher.)
+- `severity < Error` (Match anything below Error.)
+
+### Confidence
+
+Describes how confident Tiger is that a problem is real and not a false positive. If you are spammed with many false
+positives, you may try raising the required confidence level.
+
+These are the current confidence levels. They are subject to change.
+
+- `Weak` Tiger is not very confident about this report, it's quite likely to be a false positive.
+- `Reasonable` The default level.
+- `Strong` Tiger is very confident that this is a real problem.
+
+You can add the confidence trigger like so:
+
+- `confidence == Weak` (Only match weak.)
```

Comment:
Here too.

The idea was that triggers would be more familiar to modders, right? Most modders won't even know that == is a thing in pdx script.

Reply:
If we're going to treat them exactly the same, then why not drop the `==` version from the enum?